src/infrastructure/s3/s3ZipService.ts

```diff
@@ -0,0 +1,34 @@
+import { S3 } from 'aws-sdk';
+import IncorrectFileTypeError from '../../errors/IncorrectFileTypeError';
+import NoBodyError from '../../errors/NoBodyError';
+
+export default async (
+  s3: S3,
+  bucket: string,
+  folder: string | undefined,
+  adrDocumentId: string,
+): Promise<S3.Body> => {
+  const key = folder ? `${folder}/adr-documents/${adrDocumentId}.zip` : `/adr-documents/${adrDocumentId}.zip`;
+
+  console.info(`Bucket name: ${bucket}`);
+  console.info(`Item key: ${key}`);
+
+  const response = await s3
+    .getObject({
+      Bucket: bucket,
+      Key: key,
+    })
+    .promise();
+
+  if (response.ContentType !== 'application/octet-stream' && response.ContentType !== 'application/zip') {
+    // TODO: Cover this in case we encrypt the zip as octet stream or some other form
```

Comment:
have we got another ticket to handle this?

Reply:
I thought I would keep the code consistent with what was done for the existing routes, and left it in there. Happy to remove this comment and the octet-stream logic if you think it is not needed?

Reply:
Same answer as with Leo's question above, it's in all the existing routes so rather than deviate it was kept consistent.

Reply:
Looking online it's better for some browsers to use octet-streams, but as you can see none of the existing routes are using it currently.

Reply:
Given that it's going to be used by VTM and I assume you can use vtm om various browsers, I think it would be good to keep it in :)

Reply:
Fine by me, i've not had much experience with octet-streams so happy to go with the concencus of everyone here. It's not currently being used, but if that changes I would assume it is a straight forward change.